sdk/timeseriesinsights/remove-test-resources-pre.ps1

```diff
@@ -0,0 +1,53 @@
+# IMPORTANT: Use the contents of this file as a template on what resources should be cleaned in the test subscription
+
+# # Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# IMPORTANT: Do not invoke this file directly. Please instead run eng/Remove-TestResources.ps1 from the repository root.
+
+#Requires -Version 6.0
+#Requires -PSEdition Core
+
+# Use same parameter names as declared in eng/Remove-TestResources.ps1 (assume validation therein).
+[CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = 'Medium')]
+param (
+    [Parameter(Mandatory = $true)]
+    [string] $ResourceGroupName,
+
```

Comment:
extra newlines

Reply:
I believe we don't need this file. I remember @azabbasi mentioning something that we needed that for digital twins because there was an issue with deleting resources in preview.